content/init-softlayer.md

```diff
@@ -34,15 +34,29 @@ VLANs provide the ability to partition devices and subnets on the network. To or
 ---
 ## Step 2: Deploy {: #deploy }
 
-1. Install [CLI v2](cli-v2.md).
+1. Install [BOSH CLI v2](cli-v2.md), make sure to download SoftLayer specific version of bosh-cli [linux](https://bosh-softlayer-artifacts.s3.amazonaws.com/bosh-cli-5.0.1.1-softlayer-linux-amd64) or [mac](https://bosh-softlayer-artifacts.s3.amazonaws.com/bosh-cli-5.0.1.1-softlayer-darwin-amd64).
```

Comment:
I don't think we should be encouraging separate CLIs. For general OSS users, it sounded like they can use persistent IPs within IBM Cloud and have the same workflow as all other IaaSes. Was that the only reason users needed this custom `bosh-cli` binaries?

Reply:
As many users may use dynamic IP, we'd like to also support it.